speed:
    turtle_angular_speed: 0.5
